mava/components/jax/building/parameter_client.py

```diff
@@ -120,6 +120,13 @@ def on_building_executor_parameter_client(self, builder: SystemBuilder) -> None:
         else:
             set_keys = [x for x in set_keys if x.startswith("executor")]
 
+        # Set this so the evaluator is able to update continually
+        # with the latest parameters
+        if builder.store.is_evaluator is True:
+            update_period = 1
```

Comment:
Isn't 1 a bit too much? This means that we are pulling the networks at every environment step.

Reply:
It is set to 1 so that the evaluator can continually get the latest executor steps value from the parameter server [here](https://github.com/instadeepai/Mava/blob/9b768f0f1b1955ab24cc30da613918479e423db0/mava/environment_loop.py#L651). Unless there is a better way to continually update only the steps instead of all the parameters?

Reply:
We can maybe create a `force_update` function for the executor? When we call `force_update` it forces the `parameter_client` to update the executor. I am just worried that setting it to 1 would affect performance when the evaluators are running.

Reply:
Done, I added a flag named force_update that if is it true the update will happen immediately else it will be based on the update config

Reply:
After having discussion with @KaleabTessera  I find out that the hook should not include parameters so in order to keep the same design, we will add a new function called `force_update` in the executor.

Reply:
Great, thanks :)